assets/slack/manifest.yaml

```diff
@@ -34,7 +34,10 @@ types:
       writer: user | workspace#member
       parent_workspace: workspace
     permissions:
-      can_comment: commenter | writer
+      can_delete: parent->can_administer_channel
+      can_write: writer | can_delete
+      can_read: can_comment | can_write
```

Comment:
is this right? can_read and can_comment have a circular relationship?

Reply:
ugh, thanks! I'll fix that.